### ISSUE TITLE
IOS support

### ISSUE THREAD

**Issue by smx-smx** [smx-smx/ezinject#6]
Would be nice to have, but I don't have a device for testing.

Depends on #5

**Comment by zsnell02** [comment]
Corellium would be a good starting point

**Comment by smx-smx** [comment]
Isn't it a paid service?

**Comment by zsnell02** [comment]
Yes, paid by how much you use it. Actually I have an iOS device you can test with.

**Comment by smx-smx** [comment]
I *might* be able to borrow a state of the art iPod Touch 4G (not sure when I'll get the time to look at it tho). But hey better than nothing

**Comment by smx-smx** [comment]
> Actually I have an iOS device you can test with.

I just created a Discord server for ezinject if you want to discuss about it.
You can find the details in the README
